src/display/ui_state.rs

```diff
@@ -92,6 +92,7 @@ pub struct UIState {
     pub connections_map: HashMap<Connection, ConnectionData>,
     /// Used for reducing logging noise.
     known_orphan_sockets: VecDeque<LocalSocket>,
+    pub interface_name: String,
```

Comment:
I think it's better to store it as `Option<String>`, where `None` indicates "all interfaces" (it's best to add a comment for this). This is the more "rusty" way to do it.